site/topics/values-variables-types/values-variables-types.rst

```diff
@@ -66,8 +66,8 @@ Languages
    
 * The world is a screwed up, scary, place (for mathematicians, anyways). If you want to fall down this particular rabbit hole:
     * https://en.wikipedia.org/wiki/G%C3%B6del%27s_incompleteness_theorems
-	* http://en.wikipedia.org/wiki/Principia_Mathematica
-	* http://en.wikipedia.org/wiki/Computability   
+    * http://en.wikipedia.org/wiki/Principia_Mathematica
+    * http://en.wikipedia.org/wiki/Computability
```

Comment:
Hell, since we're saying the universe is scary, it turns out that the universe must be [probabalistic in nature](https://en.wikipedia.org/wiki/Bell%27s_theorem), at least with our current understandings of math.

Reply:
probably

Reply:
I actually already yeeted this in a previous PR --- I guess I just got carried away with fixing spaces/tabs.

Reply:
Wow.